ckanext/canada/schemas/dataset.yaml

```diff
@@ -56,6 +56,25 @@ sidebar_show_fields:
 #
 dataset_fields:
 
+# System Fields
+- field_name: last_sync_date
+  label:
+    en: Last Sync Date
+    fr: FR Last Sync Date FR
+  form_snippet: null
+  display_snippet: date.html
+  validators: ignore_missing ignore_not_sysadmin string_safe isodate no_future_date
+
+- field_name: out_of_sync
+  label:
+    en: Out of Sync with Portal
+    fr: FR Out of Sync with Portal FR
+  form_snippet: null
+  display_snippet: null
+  validators: ignore_missing ignore_not_sysadmin boolean_validator
+  #FIXME: output_validators not working...
+  output_validators: output_boolean_validator
```

Comment:
I wouldn't duplicate these fields in the dataset schema because updating dataset metadata is really slow. Can't you query them from your new table when you need them instead?